File: generator/helperGenerateRandomColor.m
    function color = helperGenerateRandomColor()
        % GENERATERANDOMCOLOR  Generate a random hex color.
        rgb = randi([0, 255], [1, 3]);    % Random RGB values
        color = sprintf('#%02X%02X%02X', rgb(1), rgb(2), rgb(3));  % e.g., '#D67615'
    end

File: generator/helper_generate_random_color.py
import random

def helper_generate_random_color():
    """
    Generate a random hex color.

    Returns:
        str: A hex color string, e.g., '#D67615'
    """
    rgb = [random.randint(0, 255) for _ in range(3)]
    return "#{:02X}{:02X}{:02X}".format(*rgb)
